src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -17,10 +17,10 @@
 	<script src="${rootURL}/plugin/build-pipeline-plugin/js/build-pipeline.js"></script>
 	<script id="build-card-template" type="text/x-handlebars-template">
 <table class="build-card rounded {{build.status}} {{#if project.disabled}}disabled{{/if}}">
-	<tbody>
-		<tr class="header">
+	<thead>
+		<tr>
 			<td>
-				<div class="overflow-hidden">
+				<div class="header">
```

Comment:
@oleg-nenashev Please, review the changes. It is not only a CSS class rename. The semantic is completely different.